routes/repos.js

```diff
@@ -7,79 +7,75 @@ module.exports = function ( req, res, next )
 {
     var username = req.session.user.profile.username;
 
-    var onActiveRepos = function ( err, user )
+    var onActiveRepos = function ( err, user, activeRepos )
     {
         if( err )
         {
             return next( err );
         }
 
+        var linesFound = activeRepos.reduce( ( acc, r ) => acc + r.linesFound, 0 );
+        var linesHit = activeRepos.reduce( ( acc, r ) => acc + r.linesHit, 0 );
+
         res.render( "repos", {
             layout: "layout.html",
             title: "Repos",
             repos: user.repos,
-            activeRepos: user.activeRepos,
-            authed: true } );
+            activeRepos: activeRepos,
+            linesFound: linesFound,
+            linesHit: linesHit,
+            linesPercent: 100 * linesHit / linesFound,
+            authed: true
+        } );
     };
 
     var onUserRepos = function ( err, user )
     {
-        user.save( function ( err )
+        user.repos = user.repos.sort( function ( a, b )
```

Comment:
I was not previously using them but started to on this PR. It isn't something where I want to go through and replace every instance right now.